### ISSUE TITLE
Some routes are not authenticated routes

### ISSUE THREAD

**Issue by nikolozdz** [umkc-cs-451-2020-spring/semester-project-group-5#72]
These routes are not authenticated
post '/transactions'
    get  '/transactions/:id'
    put  '/transactions/:id'
    post '/transactions/search'

    post   '/triggers'
    get    '/triggers/:id'
    put    '/triggers/:id'
    delete '/triggers/:id'
    post   '/triggers/search'

    get  'triggered-events/:id'
    post 'triggered-events/search'

**Comment by mikelhomeless** [comment]
Yeah, I noticed this when I was testing once. The problem lies in the fact that I have rails using the user_id found in the url path to verify that the user has access, but these routes don't have a user_id. I do have an Idea of how to fix this issue, I'll get back to you when I have more details.
